tests/actions.go

```diff
@@ -1037,6 +1039,10 @@ func (oa *operatorActions) getBlockWriterPod(info *TidbClusterConfig, database s
 			RestartPolicy: corev1.RestartPolicyAlways,
 		},
 	}
+	if info.OperatorTag != "e2e" {
+		pod.Spec.Containers[0].ImagePullPolicy = corev1.PullAlways
+	}
+	return pod
```

Comment:
We would always use PullAways Policy in non-e2e case.

Reply:
In which scenarios? If it's not `e2e`, the image will not be changed,  e.g. v1.1.0-rc.2.

Reply:
ah, I see. It uses `E2EImage` here.